app/models/orchestration_stack.rb

```diff
@@ -36,6 +36,7 @@ class OrchestrationStack < ApplicationRecord
   virtual_has_many :security_groups
   virtual_has_many :cloud_networks
   virtual_has_many :orchestration_stacks
+  virtual_has_many :job_plays
```

Comment:
@jntullo We should not need this relationship. Line 21 already `has_many :resources`. 
`job_plays` make senses only when the object is `ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job` which is a subclass of `OrchestrationStack`. I don't know whether it is possible to for the API to add `job_plays` at the job level. 

Anyway, right now it is safe to assume all `resources` are `job_plays`, or the SUI can make an additional filtering  `resources` to limit to jobs only.